src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -77,10 +77,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     String user = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+    String recipient = request.getParameter("recipient");
 
-    Message message = new Message(user, text);
+
+    Message message = new Message(user, text, recipient);
     datastore.storeMessage(message);
 
-    response.sendRedirect("/user-page.html?user=" + user);
+    response.sendRedirect("/user-page.html?user=" + recipient);
```

Comment:
What is the exact motivation behind this change?

Reply:
From my understanding it's so that the user returns to the page of the recipient, which is where they started from, and not their own page?

Reply:
Thanks for the explanation, that makes sense to me.